supervisor/api/resolution.py

```diff
@@ -62,3 +76,29 @@ async def dismiss_issue(self, request: web.Request) -> None:
     def healthcheck(self, request: web.Request) -> Awaitable[None]:
         """Run backend healthcheck."""
         return asyncio.shield(self.sys_resolution.healthcheck())
+
+    @api_process
+    async def enable_check(self, request: web.Request) -> None:
+        """Enable check."""
+        if request[REQUEST_FROM] != self.sys_homeassistant:
```

Comment:
Can we make this a decorator? Seems like it will be used more often. `@require_user`